lib/perl/KoL.pm

```diff
@@ -1,5 +1,3 @@
-# $Id$
```

Comment:
Why are you removing this. No one said you could!

Reply:
bruh